resources/views/bread/edit-add.blade.php

```diff
@@ -47,7 +47,7 @@ class="form-edit-add"
 
                             <!-- Adding / Editing -->
                             @php
-                                $dataTypeRows = $dataType->{(isset($dataTypeContent->id) ? 'editRows' : 'addRows' )};
+                                $dataTypeRows = $dataType->{($dataTypeContent->getKey() ? 'editRows' : 'addRows' )};
```

Comment:
Well, it could be that the primary key is not `id`.
So I think we should check for `$dataType->exists` (boolean) instead.